src/plugins/js-eval/run.js

```diff
@@ -1,7 +1,15 @@
+const { Realm, initializeAgent, FEATURES, inspect: inspectEngine262 } = require('engine262');
 const { Script, SourceTextModule, createContext } = require('vm');
 const util = require('util');
 const builtinModules = require('module').builtinModules.filter((a) => !/^_|\//.test(a));
 
+initializeAgent({
```

Comment:
Could this stuff be conditional on engine262 being selected? Maybe even don't import engine262 eagerly at all.